StripeFinancialConnections/StripeFinancialConnections/Source/API Bindings/Models/ConsumerSession/ConsumerSessionModels.swift

```diff
@@ -6,11 +6,38 @@
 //
 
 import Foundation
+@_spi(STP) import StripeCore
 
 struct ConsumerSessionData: Decodable {
     let clientSecret: String
     let emailAddress: String
     let redactedFormattedPhoneNumber: String
+    let verificationSessions: [VerificationSession]
```

Comment:
you might have done this, but worth doing a double-check that this property is non-null in server

Reply:
[required](https://stripe.sourcegraphcloud.com/git.corp.stripe.com/stripe-internal/pay-server/-/blob/lib/consumer/api/resources/auth/consumer_session_resource.rb?L23)!